src/ripple/protocol/impl/Sign.cpp

```diff
@@ -24,22 +24,22 @@ namespace ripple {
 
 void
 sign (STObject& st, HashPrefix const& prefix,
-    KeyType type, SecretKey const& sk)
+    KeyType type, SecretKey const& sk,
+    SF_Blob const& sigField)
 {
     Serializer ss;
     ss.add32(prefix);
     st.addWithoutSigningFields(ss);
-    set(st, sfSignature,
+    set(st, sigField,
         sign(type, sk, ss.slice()));
 }
 
 bool
-verify (STObject const& st,
-    HashPrefix const& prefix,
-        PublicKey const& pk,
-            bool mustBeFullyCanonical)
+verify (STObject const& st, HashPrefix const& prefix,
+        PublicKey const& pk, bool mustBeFullyCanonical,
```

Comment:
So I noticed that `mustBeFullyCanonical` is unused. We should just remove it. You can cherry-pick 248030147291f53017060b12b7d18b5172e91892 if you want.